stylesheets/desktop/stylesheet-desktop-all.css

```diff
@@ -0,0 +1,96 @@
+* {
+  margin: 0;
+  font-family: "Georgia", serif;
+}
+
+header {
+  background-color: #093C60;
+  padding: 5% 0% 5% 0%;
+  position: relative;
+}
+
+.header-intro {
+  margin-left: 10%;
+}
+
+.header-checkout {
+  position: absolute;
+  right: 0%;
+  top: 7%;
+}
+
+.header-logo {
+  width: 30%;
+}
+
+.header-title {
+  color: #81B741;
+  font-size: 1.5em;
+  width: 40%;
+}
+
+.cart-logo {
+  width: 10%;
+}
+
+#header-price {
+  color: white;
+  display: inline-block;
+}
+
+.button {
+  position:absolute;
+  right: 8%;
+  top: 20%;
+  background-color: #81B741;
+  border: none;
+  color: white;
+  padding: 1% 3%;
+  text-align: center;
+  text-decoration: none;
+  font-size: 1em;
+}
+
+.global-navigation {
+  display: flex;
+  font-size: 1em;
+  font-weight: bold;
+  background: #ECECEC;
+  padding: 2% 0% 2% 0%;
+}
+
+.nav-link {
+  color: #093C60;
+  text-decoration: none;
+  padding: 0% 1% 0 15%;
```

Comment:
I think you want to use margins and not padding here, they are similar but can produce different results. Your nav-link is hiding behind nav-link-2 at some resolutions. using margins instead of padding should fix this.